cibyl/sources/elastic/filter.py

```diff
@@ -0,0 +1,142 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+import argparse
+import logging
+from cibyl.sources.elastic.client import ElasticSearchClient
+from sys import exit
+
+LOG = logging.getLogger(__name__)
+
+
+class ElasticSearchOSPFilter:
+
+    def __init__(self: object, elastic_client: object) -> None:
+        self.es = elastic_client
+
+    def get_job_by_name(self: object, query_data: dict):
+        """Get all the job information with an exact match
+
+        :param query_data: Information required for the query:
+                Index, Search key and Search Value.
+        :type query_data: dict
+        """
+        query_body = {
+           'query': {
+                'term': {
+                    f"{query_data['key']}.keyword": {
+                        'value': query_data['value']
+                    }
+                }
+            }
+        }
+        hits = self.__query_get_hits(query_data['index'], query_body)
+        print(hits)
```

Comment:
Do we have a rule on when to use the log and when to print? It is something I have also faced before.

Reply:
Sources should not print anything. There is a separate component in Cibyl that will do the printing. Source should only log information and higher-level probably than "info" level

Reply:
Good to know. I've removed these print statements. Thanks.